src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -104,7 +104,7 @@ private void testGeneralQuestionOperations() throws Exception {
         testAjaxOnVisibilityMessageButton();
 
         testDeleteQuestionAction(2);
-        testDeleteQuestionAction(1);
+        testEditThenDeleteQuestionAction(1);
```

Comment:
Nice way of combining tests so we don't spend (too much) more resources!

Good workaround since it is too difficult at this point in time to test this at the Javascript level (which we should actually).